test/system/250-systemd.bats

```diff
@@ -125,4 +125,33 @@ function service_cleanup() {
     service_cleanup
 }
 
+@test "podman generate - systemd - template" {
+    cname=$(random_string)
+    run_podman create --pull=always --name $cname $IMAGE top
```

Comment:
Also add a test with `--name=$cname`. And please remove `--pull=always` everywhere, this is not needed and makes the test unnecessary slow.